libs/ui/src/ui_strings/keyboard_shortcut_handlers.tsx

```diff
@@ -4,6 +4,26 @@ import { useAvailableLanguages } from '../hooks/use_available_languages';
 import { useLanguageControls } from '../hooks/use_language_controls';
 import { useAudioControls } from '../hooks/use_audio_controls';
 
+export type AccessibilityCommands =
+  | 'toggle-audio'
+  | 'replay'
+  | 'decrease-playback-rate'
+  | 'increase-playback-rate'
+  | 'toggle-pause'
+  | 'decrease-volume'
+  | 'increase-volume';
+
+export const ACCESSIBILITY_COMMAND_KEYS: Record<AccessibilityCommands, string> =
```

Comment:
nit: I think this is sort of re-implementing `enum` functionality - these two defs could be consolidated into 
```ts
export enum AccessibilityCommandKeys {
  TOGGLE_AUDIO = 'M',
  REPLAY = 'R',
  // etc...
}
```

Reply:
[b549647](https://github.com/votingworks/vxsuite/pull/4676/commits/b549647e3759424a4e5805e63681bec5318949ae)